eo-parser/src/test/java/org/eolang/parser/XMIRTest.java

```diff
@@ -127,7 +127,9 @@ void convertsAntlrToEbnf(@TempDir final Path temp) throws Exception {
             Matchers.allOf(
                 Matchers.containsString("<byte> := /[0-9A-F]/ /[0-9A-F]/ \\\\"),
                 Matchers.containsString("<string> := 'DQ' {"),
-                Matchers.containsString("<text> := 'DQ' 'DQ' 'DQ' [")
+                Matchers.containsString("<text> := 'DQ' 'DQ' 'DQ' ["),
+                Matchers.containsString("\\textvisiblespace{}"),
+                Matchers.containsString("\\textquotesingle{}")
```

Comment:
@yegor256 I think it would be better add more specific matcher here. We should check not only that "string contains `\\textquotesingle{}`", but that `"'"` is successfully replaced with `\\textquotesingle{}`. WDYT?

Reply:
@maxonfjvipon this is true, but such a test would require a lot more efforts. We will better do it later, when we create a new Maven plugin for this kind of mapping from .g4 to .txt: https://github.com/objectionary/eo/issues/2464

Reply:
@yegor256 maybe add todo for it then?

Reply:
@maxonfjvipon I thought about this, but it doesn't make sense to have a puzzle in this repository, since the entire test will be moved out soon (https://github.com/objectionary/eo/issues/2464)